transform.go

```diff
@@ -79,19 +80,23 @@ func opTypeToString(value OpType) string {
 	case OpTypeIn:
 		return "in"
 	}
-
 	return "??unknown??"
 }
 
 type OpNode struct {
 	BucketIdx BucketID
 	Op        OpType
+	OpNot     bool
```

Comment:
This is from the old implementation of NotExpr, which is already completed

Reply:
Removed